docs: default feature of `runtime-check-and-print` is left out

@mkovaxx  I get this error when opening to project.

```sh
❯ cargo check
    Checking anodized v0.3.0 (/Users/chammika_mannakkara/misc/anodized/anodized/crates/anodized)
error[E0080]: evaluation panicked: anodized: a runtime feature must be selected:
              `runtime-check-and-panic`
              `runtime-check-and-print`
              `runtime-no-check`
  --> crates/anodized/src/lib.rs:28:5
   |
28 | /     panic!(
29 | |         r#"anodized: a runtime feature must be selected:
30 | | `runtime-check-and-panic`
31 | | `runtime-check-and-print`
32 | | `runtime-no-check`"#
33 | |     )
   | |_____^ evaluation of `BACKEND` failed here

For more information about this error, try `rustc --explain E0080`.
error: could not compile `anodized` (lib) due to 1 previous error
```

Due to commented code here in `anodized/Cargo.toml` is there any reason for leaving out this default feature ? 

```rust
[features]
# default = ["runtime-check-and-print"]
```

--- a/crates/anodized/Cargo.toml
+++ b/crates/anodized/Cargo.toml
@@ -14,6 +14,8 @@ license.workspace = true
 proc-macro = true
 
 [features]
+# Users should explicitly select a runtime feature when using this crate.
+# The default below is for development only:
 # default = ["runtime-check-and-print"]
 runtime-check-and-panic = []
 runtime-check-and-print = []

